test/TestPercentageMath.sol

```diff
@@ -164,12 +167,9 @@ contract TestPercentageMath is Test {
         uint256 y,
         uint16 percentage
     ) public {
-        vm.assume(percentage <= PERCENTAGE_FACTOR);
-        // prettier-ignore
-        vm.assume(
-            (percentage == 0 || y <= MAX_UINT256_MINUS_HALF_PERCENTAGE / percentage) &&
-            (PERCENTAGE_FACTOR - percentage == 0 || x <= MAX_UINT256_MINUS_HALF_PERCENTAGE / (PERCENTAGE_FACTOR - percentage))
-        );
+        vm.assume(percentage <= 1e4);
+        vm.assume(percentage == 0 || y <= (type(uint256).max - 0.5e4) / percentage);
+        vm.assume(1e4 - percentage == 0 || x <= (type(uint256).max - y * percentage - 0.5e4) / (1e4 - percentage));
```

Comment:
at the opposite of what is done elsewhere

Reply:
you mean the literal values instead of using the constants ?